crates/zed/src/languages/clojure.rs

```diff
@@ -0,0 +1,132 @@
+use anyhow::{anyhow, bail, Context, Result};
+use async_trait::async_trait;
+pub use language::*;
+use lsp::LanguageServerBinary;
+use smol::fs::{self, File};
+use std::{
+  any::Any,
+  env::consts,
+  path::PathBuf,
+};
+use util::{
+  fs::remove_matching,
+  github::{latest_github_release, GitHubLspBinaryVersion}
+};
+
+#[derive(Copy, Clone)]
+pub struct ClojureLspAdapter;
+
+#[async_trait]
+impl super::LspAdapter for ClojureLspAdapter {
+    async fn name(&self) -> LanguageServerName {
+        LanguageServerName("clojure-lsp".into())
+    }
+
+    fn short_name(&self) -> &'static str {
+        "clojure"
+    }
+
+    async fn fetch_latest_server_version(
+        &self,
+        delegate: &dyn LspAdapterDelegate,
+    ) -> Result<Box<dyn 'static + Send + Any>> {
+        let release =
+            latest_github_release("clojure-lsp/clojure-lsp", false, delegate.http_client())
+                .await?;
+        let platform = match consts::ARCH {
+            "x86_64" => "amd64",
+            "aarch64" => "aarch64",
+            other => bail!("Running on unsupported platform: {other}"),
+        };
+        let asset_name = format!("clojure-lsp-native-macos-{platform}.zip");
```

Comment:
is that hard to support linux as well? it should follow the same standard

Reply:
Zed is macos only for now

Reply:
Zed is macOS only for now (see https://github.com/zed-industries/zed/issues/5395)